playground/tsconfig.json

```diff
@@ -1,4 +1,16 @@
 {
   "extends": "../tsconfig.common.json",
+  "compilerOptions": {
+    "lib": ["ES2020", "DOM", "DOM.Iterable"],
+    "useDefineForClassFields": true,
+    "module": "ESNext",
+
+    /*Bundler Mode */
+    "allowImportingTsExtensions": true,
+    "moduleResolution": "bundler",
+    "isolatedModules": true,
+    "noEmit": true,
```

Comment:
emitting error would be better to catch wrong types

Reply:
I was betting on the IDE hahaha 
Considering DX, I believe that the IDE will manage most of the scenarios. If that's not sufficient, perhaps we could use:
https://vite-plugin-checker.netlify.app/introduction/introduction.html

Reply:
this point reminded me to add type typing at build time. 🤣